src/number_theory/bernoulli.lean

```diff
@@ -257,3 +257,119 @@ begin
   { ext x, rw bernoulli_eq_bernoulli' (succ_ne_zero x ∘ succ.inj) },
   { ring },
 end
+
+
+theorem bernoulli_power_series:
+  (exp ℚ - 1) * power_series.mk (λ n, (bernoulli n / nat.factorial n)) = X :=
+ begin
+   sorry,
+ end
+
+section faulhaber
+
+lemma aux_cauchy_prod (n:ℕ):
+mk (λ (p : ℕ), bernoulli p / ↑(p.factorial)) *
```

Comment:
I think `open_locale nat` or similar will let you use `!` for  factorial, which might make things easier to read.

Reply:
done, in all places.